### DISCUSSION TITLE
How to write one row to Google Sheet

### DISCUSSION THREAD

**Discussion by SuperTempAccount** [AutomaApp/automa#1419] [Q&A]
Hey Automaers,

How can I write a single value to Google Sheets?
I'm trying to write the last value in a single-column table to a Google Sheet.

I am writing to: `sheet1!C2:C2`

The `Google Sheet` block has two options: `Table` for writing the whole table, or `Custom` for a custom JSON value.

I've selected `Custom` and entered this for the value.
<img width="499" alt="image" src="https://github.com/AutomaApp/automa/assets/83566267/a7a84170-7ff4-462e-be28-b1484127e1f0">

I get the error `t.forEach is not a function`.

I tried including multiple values in the JSON as a test but still receive the same error.
<img width="499" alt="image" src="https://github.com/AutomaApp/automa/assets/83566267/74e0f5bb-cab0-4802-9705-e9858a32c8a5">

**Comment by juliocfc92** [comment]
![image](https://github.com/user-attachments/assets/43aefe8d-5014-4a0f-bda2-357e79b2818c)
